src/main/java/frc/robot/subsystems/PhotonSubsystem.java

```diff
@@ -224,7 +224,12 @@ public void periodic() {
       //calculate yaw
       Rotation2d yaw = Rotation2d.fromDegrees(target.getYaw()*-1);
       //calculate range
-      double range = (range(target.getPitch())-1)*1.15+1;
+      double range = (range(target.getPitch()));
+      if (range < 6 & range > 1.5){
```

Comment:
Use &`& `not `&`. &`& `compares booleans but `& `does a bitwise AND which is unpredictable what it will do in this case.

Reply:
bitwise AND might technically work but its not a good practice. Ideally use `&&`